scheme/proto/src/main/java/cz/o2/proxima/scheme/proto/ProtoSerializerFactory.java

```diff
@@ -171,4 +197,62 @@ public MessageT fromJsonValue(String json) {
       return (AttributeValueAccessor<MessageT, OutputT>) accessor;
     }
   }
+
+  @VisibleForTesting
+  static class TransactionProtoSerializer<T> implements ValueSerializer<T> {
+
+    @SuppressWarnings("unchecked")
+    static <V> TransactionProtoSerializer<V> ofTransactionClass(String className) {
+      switch (className) {
+        case "cz.o2.proxima.transaction.Request":
+          return (TransactionProtoSerializer<V>)
+              new TransactionProtoSerializer<>(
+                  new ProtoValueSerializer<>(ProtoRequest.class.getName()),
+                  req -> ProtoRequest.newBuilder().build(),
```

Comment:
These translation will have to be updated when we have actual fields in `Request` and `Response`.